.pipelines/templates/template-job-deploy-azure-env.yml

```diff
@@ -58,7 +45,7 @@ jobs:
             displayName: Download Deployer
           - template: ./template-deploy-azure-env.yml
             parameters:
-              configDirectory: $(System.ArtifactsDirectory)/config/drop/deploy
+              configDirectory: $(Agent.BuildDirectory)/s/deploy
```

Comment:
According to the [documentation](https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/multi-repo-checkout?view=azure-devops#checkout-path) when doing single checkout the code is stored in `$(Agent.BuildDirectory)/s`  which is the root of the pulled repository. The `deploy/` [folder](https://msazure.visualstudio.com/AzureRedHatOpenShift/_git/RP-Config?path=%2Fdeploy) stores the config files which are used to deploy the RP.